shared/fpga_xilinx.core

```diff
@@ -0,0 +1,18 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:ibex:fpga_xilinx_shared"
+description: "Collection of useful RTL for Xilinx based examples"
+filesets:
+  files_sv:
+    files:
+      - rtl/fpga/xilinx/prim_clock_gating.sv
+      - rtl/fpga/xilinx/clkgen_xil7series.sv
+      - rtl/ram_1p.sv
```

Comment:
I'm not a huge fan of including the same file in multiple core files (that's what dependencies are for), but good enough for now.